homeassistant/components/pyload/sensor.py

```diff
@@ -158,13 +161,20 @@ def __init__(
         entry_id: str,
     ) -> None:
         """Initialize a new pyLoad sensor."""
-        self._attr_name = f"{client_name} {entity_description.name}"
+        self._attr_name = f"{entity_description.name}"
         self.type = entity_description.key
         self.api = api
         self._attr_unique_id = f"{entry_id}_{entity_description.key}"
         self.entity_description = entity_description
         self._attr_available = False
         self.data: StatusServerResponse
+        self.device_info = DeviceInfo(
+            manufacturer="PyLoad Team",
+            model="pyLoad",
+            configuration_url=self.api.api_url,
+            identifiers={(DOMAIN, entry_id)},
+            name=client_name,
```

Comment:
Can be omitted, primary devices without name take the config entry title by default

Reply:
oh and i forgot entry type service